### PR TITLE
resolve #2 write battery parameters

### LINKED ISSUE #2: Write battery parameters
Hey, really enjoyed finding your project in amongst a few others, (from a bit too scriptey, to a bit too objectey.)
I really enjoy the abstraction you've done and the command line utility really shows off how your implementation is straight forward.

In the readme, it's stated "Write battery parameters this feature is a work in progress".

I have a Tracer 2206AN and have already written my values for a 12V 100Ah battery. It seems I chose values close to those in your example.

I've raised this issue to cover adding that functionality. Would you mind if I added that functionality and created a PR for you?

### PR DESCRIPTION
**Pull request by hrford**
I placed the write_battery_params.py in an example folder and used setup.py to re-install this version.
Should we be increasing the version number as we add features?

(One of the added features might be tweaked when I add a temperature settings)

ps, the try/except works well with minimalmodbus, this is actually catching an error from a real piece of hardware! :)

### CODE REVIEW

File: epevermodbus/driver.py [hunk 1]
```diff
@@ -13,6 +14,21 @@ class EpeverChargeController(minimalmodbus.Instrument):
 
     """
 
+    battery_voltage_control_register_names = [
+        "over_voltage_disconnect_voltage",
+        "charging_limit_voltage",
+        "over_voltage_reconnect_voltages",
```
Comment (rosswarren) [inline epevermodbus/driver.py:20]: is this a typo here? Should it be `over_voltage_reconnect_voltage`?

File: epevermodbus/driver.py [hunk 2]
```diff
@@ -69,7 +93,7 @@ def get_load_power(self):
 
     def get_battery_current(self):
         """Battery current in amps"""
-        return self.retriable_read_long(0x331B, 4) / 100
```
Comment (rosswarren) [inline epevermodbus/driver.py:72]: good catch 👍

File: examples/write_battery_params.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
+from minimalmodbus import SlaveReportedException
+
+from epevermodbus.driver import EpeverChargeController
+
+
+def main():
```
Comment (rosswarren) [inline examples/write_battery_params.py:6]: this example script looks useful 👍

File: epevermodbus/driver.py (hunk 1 above)
Comment (hrford) [reply epevermodbus/driver.py:20]: It should! Nice catch.

File: epevermodbus/driver.py (hunk 2 above)
Comment (hrford) [reply epevermodbus/driver.py:72]: I'm glad you ... signed off on this one ;)

**Review by rosswarren — APPROVED** [review]

**Pull request merged**